vector/src/main/res/values/strings_tchap.xml

```diff
@@ -72,6 +72,9 @@
         <item quantity="other">History limited to %d days</item>
     </plurals>
     <string name="tchap_room_invited_you">%s invited you</string>
+    <string name="tchap_fab_new_direct_message">New direct message</string>
+    <string name="tchap_fab_new_room_forum">New room/forum</string>
```

Comment:
Please use String coming from Element `fab_menu_create_room` instead of creating a new one and update the value to `New room/forum`

Remove the old String ressource `tchap_room_creation_title`